www/source/javascripts/community.js

```diff
@@ -1,19 +1,128 @@
-var community_events = function(data) {
-    if ($(".community-events")) {
-        if (Array.isArray(data)) {
-            data.forEach(function(e) {
-                var ev = "<p>";
-                ev += e.title;
-                ev += "<br>";
-                ev += e.link;
-                ev += "<br>";
-                ev += e.pub_date;
-                ev += "<br>";
-                ev += e.description;
-                ev += "</p>";
-
-                $(".community-events").append(ev);
-            });
-        }
+$(function() {
+  var htmlDecode = function(input) {
```

Comment:
Here is an example of what comes back in the `guid` field: `http://events.chef.io/?post_type=event&#038;p=485`.  I was using this to convert the `&#038;` into an actual `&`.